projects/unrar/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "http://www.rarlab.com/"
+primary_contact: "roshal@rarlab.com"
+sanitizers:
+  - address
+  - memory:
+     experimental: True
```

Comment:
Note that these wont be autofiled then. If code is simple and does not depend on system libs, remove this line.

Reply:
Done